gui/_progressbar.scss

```diff
@@ -8,43 +8,61 @@
   height: 15px;
   border: 1px solid var(--button-border-color);
   border-radius: var(--border-radius);
-  box-shadow: inset 0 0 2px #fff, 0 0 2px #aaa;
+  box-shadow: inset 0 0 0 1px #f3f3f388, 0 0 0 1px #eaeaea88;
   margin: 2px 0;
   overflow: hidden;
-  background: linear-gradient(
-      to right,
-      rgba(0, 0, 0, 0.1),
-      transparent 20%,
-      transparent 80%,
-      rgba(0, 0, 0, 0.1)
+  background: radial-gradient(
```

Comment:
I can't really verify but for the marquee, it seems the color is too pale after this change. If you could share a screenshot of the actual marquee that looks like this, I can put my mind at ease 🙂

<img width="456" alt="image" src="https://github.com/khang-nd/7.css/assets/55619201/20047cee-7bb6-4163-86d4-69ec09b3a08f">

Reply:
You are right, the colour is too pale. I'll fix it tomorrow.

Context | Image
:-:|:-: 
From Copy File Dialog | ![ProgressBarMarquee_CopyFileDialogue](https://github.com/khang-nd/7.css/assets/20926690/65e0253e-51d9-4cbe-8ef8-abd2e8dd6823)
From Windows Update | ![ProgressBarMarquee_WindowsUpdate](https://github.com/khang-nd/7.css/assets/20926690/2e3886ba-c5d3-40a7-91b2-44ebc190c576)

Also, I've just noticed that the shiny/reflective part at the top of the progress bar doesn't stretch when the entire bar is stretched vertically, so I'll fix it too.

![ae fae4w afae fae faef](https://github.com/khang-nd/7.css/assets/20926690/f521fb4f-ea0a-48ff-a751-52ef9953be9b)

Reply:
Awesome, cheers mate 🙂